docs/infrastructure/eks-quickstart.md

```diff
@@ -158,7 +156,10 @@ and copy the outputed value. To connect to the dashboard run
 kubectl proxy
 ```
 
-and then connect to [the dashboard url](http://127.0.0.1:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:https/proxy),
+and then connect to the dashboard URL
+
+<http://127.0.0.1:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:https/proxy>
```

Comment:
This makes it easier to understand that the user must open this URL and not just <http://127.0.0.1:8001> as shown in the output of the `kubectl proxy` command.